systemtest/aggregation_test.go

```diff
@@ -38,7 +38,7 @@ import (
 
 func TestTransactionAggregation(t *testing.T) {
 	systemtest.CleanupElasticsearch(t)
-	srv := apmservertest.NewUnstartedServerTB(t)
+	srv := apmservertest.NewUnstartedServer()
```

Comment:
If the test fails part way through, the server will be left running won't it? The idea behind NewUnstartedServerTB is that it should always close the server that it starts, when the test ends.

Changing NewUnstartedServerTB to NewUnstartedServer means transferring the responsibility of closing the server back to each test - that seems a bit error prone to me.

Could we change `Server.Close` to be safe to call multiple times, including concurrently?

Reply:
Good catch! Updated!